### ISSUE TITLE
"Cannot read property 'map' of undefined" when using Swagger CLI plugin

### ISSUE THREAD

**Issue by some-user123** [nestjs/nest#6387]
## Current behavior
I'm trying to add the swagger CLI plugin to an existing project (to get rid of the redundant `@ApiProperty`).

I'm using NestJs in standalone mode but use yarn workspaces to have the frontend in the same repo next to the backend.

`yarn workspace api start` (or `build` as well) work great, until I update my `nest-cli.json` to look like this:

```json
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "plugins": [
      "@nestjs/swagger"
    ]
  }
}
```

If I know execute `yarn workspace api start` it will start, but the plugin is not working. I.e. properties which are not decorated don't show up in the SwaggerUI.

If I execute `yarn workspace api build` (which deletes the `dist` output directory), it will fail:
```bash
$ yarn workspace api build
$ rimraf dist
$ nest build

 Error  Cannot read property 'map' of undefined

Done in 7.07s.
```

Now, the execution of `yarn workspace api start` will fail as well with the same error.

## Expected behavior

The project should still compile and the plugin should do its job ;-)

If something is wrong, I'd expect a more elaborate error message. I'm not an NestJs expert by far. But I'd hope that a stack trace would give a little more information, about what is going on here.

## Environment

<pre><code>
Nest version: 7.6.5 and 7.6.12

 
For Tooling issues:
- Node version: 14.5.0  
- Platform: Windows 

Others:
- yarn: 1.22.10
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by some-user123** [comment]
Sure: `https://github.com/some-user123/nest-issue-6387.git`

Thanks a lot for looking at it!

Still, I would find it very helpful to be able to debug the issue myself. So, if you have any hints how to get a more detailed error message, I'd be happy :)

**Comment by jmcdo29** [comment]
Hmm, only `yarn` gives the error. `pnpm` and `npm` both build the project without any issue. I'll try to look into why that is.

**Comment by some-user123** [comment]
Really strange. I just reproduced to problem:
```bash
git clone https://github.com/some-user123/nest-issue-6387.git
cd nest-issue-6387
yarn install --frozen-lockfile
yarn workspace api build
```

Do you have ideas/suggestions how I could further analyze the issue on my end?

**Comment by kamilmysliwiec** [comment]
![image](https://user-images.githubusercontent.com/23244943/107772127-14d68300-6d3c-11eb-86d9-efdac959d57b.png)

This issue was coming from the underlying TS compiler. Fixed in `@nestjs/swagger@4.7.13`
